functions_framework_builder/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: functions_framework_builder
-version: 0.3.0
+version: 0.3.1-dev
```

Comment:
Don't forget to bump the changelog, too....but we can do that later!

Reply:
> Don't forget to bump the changelog, too....but we can do that later!

I did bump it ... from `0.3.1-dev` to `0.3.1`. Are you talking about something else (the builder package maybe)?